crowd-server/FatemeNikkhah/myproject/api/views.py

```diff
@@ -0,0 +1,19 @@
+from rest_framework import serializers
+from rest_framework.response import Response
+from rest_framework.decorators import api_view
+from food.models import Question
+from .serializers import QuestionSerializer,ResponseSerializer
+
+
+@api_view(['GET'])
+def get_food_fact(request):

```

Comment:
When you are using DRF, you can use its great views for this like GenericView or ApiView or ...